code/modules/surgery/organs/augments_external.dm

```diff
@@ -11,41 +11,41 @@
 	name = "chest"
 	desc = "A Robotic chest"
 	icon_state = "chest"
-	max_damage = 200
+	max_damage = INFINITY
```

Comment:
oh, that's kind of annoying, I'll look into redoing it to be based off of other shit